ui/apps/platform/src/Containers/NetworkGraph/components/StyleNode.tsx

```diff
@@ -24,12 +24,17 @@ import {
 } from '@patternfly/react-topology';
 import DefaultIcon from '@patternfly/react-icons/dist/esm/icons/builder-image-icon';
 import AlternateIcon from '@patternfly/react-icons/dist/esm/icons/regions-icon';
-import FolderOpenIcon from '@patternfly/react-icons/dist/esm/icons/folder-open-icon';
-import BlueprintIcon from '@patternfly/react-icons/dist/esm/icons/blueprint-icon';
-import PauseCircle from '@patternfly/react-icons/dist/esm/icons/pause-circle-icon';
+// import { PficonNetworkRangeIcon } from '@patternfly/react-icons';
```

Comment:
Did you leave this commented because you're going to need it again soon, or can the line just be deleted?

Reply:
yep, i'll have another PR out for the externally connected decorator soon